<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="1.354" timestamp="2024-12-20T17:24:30.429289+05:30" hostname="LAP-55355"><testcase classname="" name="src.app.controllers.users.handlers" time="0.000"><error message="collection failure">ImportError while importing test module 'C:\Users\pjalodiya\PycharmProjects\Asset-Management\src\app\controllers\users\handlers.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\..\..\..\.venv\Lib\site-packages\_pytest\python.py:493: in importtestmodule
    mod = import_path(
..\..\..\..\..\.venv\Lib\site-packages\_pytest\pathlib.py:587: in import_path
    importlib.import_module(module_name)
C:\Program Files\Python313\Lib\importlib\__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1387: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1360: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:1331: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:935: in _load_unlocked
    ???
..\..\..\..\..\.venv\Lib\site-packages\_pytest\assertion\rewrite.py:184: in exec_module
    exec(co, module.__dict__)
handlers.py:14: in &lt;module&gt;
    from src.app.controllers.users.handlers import UserHandler
E   ImportError: cannot import name 'UserHandler' from partially initialized module 'src.app.controllers.users.handlers' (most likely due to a circular import) (C:\Users\pjalodiya\PycharmProjects\Asset-Management\src\app\controllers\users\handlers.py)</error></testcase></testsuite></testsuites>